internal/app/app_test.go

```diff
@@ -36,9 +35,6 @@ func TestNewPolicyAutomationApp(t *testing.T) {
 	if paApp.out == nil {
 		t.Fatalf("policyAutomationApp output is nil")
 	}
-	if !reflect.DeepEqual(paApp.out.w, io.Discard) {
```

Comment:
It would be nice to add some other test instead *(i.e. to check if ConsoleResultCollector is created by default)*